glustercli/cmd/volume.go

```diff
@@ -432,6 +434,28 @@ var volumeStatusCmd = &cobra.Command{
 	},
 }
 
+var volumeSizeCmd = &cobra.Command{
+	Use:   "size",
+	Short: helpVolumeSizeCmd,
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		volname := cmd.Flags().Args()[0]
+		vol, err := client.VolumeStatus(volname)
+		if err != nil {
+			if verbose {
+				log.WithFields(log.Fields{
+					"error": err.Error(),
+				}).Error("error getting volume size")
+			}
+			failure("Error getting Volume size", err, 1)
+		}
+		fmt.Println("Volume :", volname)
```

Comment:
You can remove the extra space before `:` but retain the one after it.

Here and below.

Reply:
done